src/files/files.module.ts

```
@@ -0,0 +1,7 @@
+import { Module } from '@nestjs/common';
+import { FilesController } from './files.controller';
+
+@Module({
+  controllers: [FilesController]
```

ESLint

[error] Insert `,`

(prettier/prettier)